### DISCUSSION TITLE
Change brightness in HyperHDR?

### DISCUSSION THREAD

**Discussion by DariBer** [awawa-dev/HyperHDR#347] [Q&A] [answered]
When I watch TV and it's dark in the room, I find the light to be to bright, so is there anyway to dim/decrease the brightness in HyperHDR

**Comment by awawa-dev** [comment, marked as answer]
Hi
You can increase RGB gammas from default 1.5 in the processing tab (it's simply exponentiation correction, won't affect max brightness) or lower max brightness/luminance (option depending if you have a classic HyperHDR calibration enabled or not, it should affect maximum brightness)

**Comment by exhibitO** [comment]
@awawa-dev  I see your reply above, classic Hyperion has a setting called Brightness gain, do you know how I can increase it in HyperHDR?

**Reply by awawa-dev** [reply]
Image processing->color adjustment. Try lower gammas from default 1.5 or increase luminescence (depends on classic calibration option)

**Reply by awawa-dev** [reply]
Best option for LEDs (gamma 1.5 is darker and not neutral, neutral is 1):
![obraz](https://github.com/user-attachments/assets/105b2207-2e73-4606-bf5e-d3241bd67bb9)

Not so good option (luminescence above 1):
![obraz](https://github.com/user-attachments/assets/66bd2b65-d9a7-4798-9d0c-a419c78f08b9)
